packages/parity-electron/src/checkClockSync.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+const { time } = require('sntp');
+
+export const MAX_TIME_DRIFT = 10000; // milliseconds
+
+export const checkClockSync : () => Promise<{isClockSync: boolean, timeDrift: number}> = () => {
```

Comment:
The syntax:
```javascript
export const checkClockSync = (): Promise<{
  isClockSync: boolean;
  timeDrift: number;
}> => {...}
```
is slight shorter.

Or even shorter:
```javascript
export const checkClockSync = async () => {
  const { t: timeDrift }: { t: number } = await time();
  return {
    isClockSync: timeDrift < MAX_TIME_DRIFT,
    timeDrift
  };
};
```
and let type inference do the job.